src/qoi/include/grins/qoi_base.h

```diff
@@ -78,7 +78,9 @@ namespace GRINS
      * Method to allow QoI to cache any system information needed for QoI calculation,
      * for example, solution variable indices.
      */
-    virtual void init( const GetPot& input, const MultiphysicsSystem& system );
+    virtual void init( const GetPot& input,
+                       const MultiphysicsSystem& system,
+                       unsigned int qoi_num );
```

Comment:
In this case I'm not talking about move-vs-copy, I'm talking about override.  The ParsedBoundary QoI was a separate development thread, so it didn't get the API change that is in this thread, so although git thought rebasing off master went without a hitch, it actually created a runtime error.  override would have caught the mistake at compile-time.

Or libmesh_override, naturally.  See https://github.com/libMesh/libmesh/issues/655 if you want to chime in on that.